packages/data/src/index.js

```diff
@@ -20,7 +20,11 @@ import isShallowEqual from '@wordpress/is-shallow-equal';
  */
 import registerDataStore from './store';
 
-export { loadAndPersist, withRehydration } from './persist';
+export {
+	loadAndPersist,
+	setPersistenceStorage,
```

Comment:
`setPersistenceStorage` - it probably should be another PR. I added it because it wasn't exposed.